resources/views/_snippets/styles.blade.php

```diff
@@ -64,19 +63,21 @@ function toggleTheme() {
     window.onload = loadTheme;
 </script>
 
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/highlight.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/languages/bash.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/languages/php.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/languages/yaml.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/languages/diff.min.js"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.9.0/languages/json.min.js"></script>
-
-<script>
-    document.addEventListener('DOMContentLoaded', (event) => {
-        document.querySelectorAll('pre code.language-bash, pre code.language-php, pre code.language-yaml, pre code.language-diff, pre code.language-json').forEach((element) => {
-            hljs.highlightElement(element);
+@isset ($codeMirror)
```

Comment:
@TomasVotruba this seems make detail rule doesn't have code highlight, see https://getrector.com/rule-detail/covers-annotation-with-value-to-attribute-rector

![Screenshot 2025-04-29 at 08 28 50](https://github.com/user-attachments/assets/b5389a1d-d95d-4155-b149-5478f7ba288a)

Reply:
Resolved at PR:

- https://github.com/rectorphp/getrector-com/pull/2987